boards/nucleo_wl55jc.json

```diff
@@ -0,0 +1,46 @@
+{
+  "build": {
+    "core": "stm32",
+    "cpu": "cortex-m4",
+    "extra_flags": "-DSTM32WL55JC1",
+    "f_cpu": "48000000L",
+    "mcu": "stm32wl55jc",
+    "product_line": "STM32WL55xx",
+    "variant": "STM32WL55JCx/WL55JC(1-2)"
+  },
+  "debug": {
+    "default_tools": [
+      "stlink"
+    ],
+    "jlink_device": "STM32WL55JC",
+    "onboard_tools": [
+      "stlink"
+    ],
+    "openocd_target": "stm32wlx",
+    "svd_path": "STM32WL5x_CM4.svd"
+  },
+  "frameworks": [
+    "arduino",
+    "cmsis",
+    "mbed",
+    "stm32cube",
+    "libopencm3",
+    "zephyr"
+  ],
```

Comment:
Please review the framework list.
- It seems there is no support for this board in the latest stable Arduino v2.0, probably will be added in v2.1.
- CMSIS and STM32Cube are not supported at the moment as well.
- libOpenCM3 doesn't have support for the `WL` series.

So I guess the only option left is Zephyr.

Reply:
Removed everything except Mbed, and Zephyr.

Reply:
It seems mbed is not available as well. At least in v6.9 which we support at the moment. I think it's reasonable to remove it for now, until the latest mbed is added to the platform.

Reply:
Removed. I am using the board successfully with Mbed 6.9, so I did not even think to check if it was officially supported.